models/protocol_tests/generate.go

```diff
@@ -266,6 +290,41 @@ func (i *testCase) TestCase(idx int) string {
 	return buf.String()
 }
 
+func mapInstantiate(m map[string]interface{}) string {
```

Comment:
How about rename to `serializeMap` since Instantiate implies creating a map, but its really creating a string serialized version of a map.